bucket/office-tool-plus.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "8.3.10.7",
+    "description": "A powerful and useful tool for Office deployments",
+    "homepage": "https://otp.landian.vip/",
+    "license": "GPL-3.0-or-later",
+    "url": "https://github.com/YerongAI/Office-Tool/releases/download/v8.3.10.7/Office_Tool_with_runtime_v8.3.10.7.7z",
+    "hash": "757ac90af36ef1e92501429172a7670b884c1472be4eab99d42eb34de5a712a0",
+    "extract_dir": "Office Tool",
+    "shortcuts": [
+        [
+            "Office Tool Plus.exe",
+            "Office Tool Plus"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/YerongAI/Office-Tool"
+    },
+    "autoupdate": {
+        "url": "https://github.com/YerongAI/Office-Tool/releases/download/v$version/Office_Tool_with_runtime_v$version.7z",
+        "hash": "$basename.*?$sha256"
```

Comment:
```suggestion
        "hash": {
            "url": "https://github.com/YerongAI/Office-Tool/releases/latest",
            "regex": "(?sm)$basename.*?$sha256"
        }
```

Reply:
This `autoupdate.hash` is actually not valid. (although PR check passed)
I will try to fix PR check later.

Result of checkver.ps1:
```
> .\checkver -f office-tool-plus
office-tool-plus: 8.3.10.7 (scoop version is 8.3.10.7)
Forcing autoupdate!
Autoupdating office-tool-plus
Downloading Office_Tool_with_runtime_v8.3.10.7.7z to compute hashes!
Office_Tool_with_runtime_v8.3.10.7.7z (47.3 MB) [=============================================================] 100%
Computed hash: 757ac90af36ef1e92501429172a7670b884c1472be4eab99d42eb34de5a712a0
Writing updated office-tool-plus manifest
```

Reply:
Thanks!